chroma-manager/agent-bootstrap-script.template

```diff
@@ -127,28 +133,39 @@ def reg_manager():
 
 def create_repo():
     tmp = tempfile.NamedTemporaryFile()
-    tmp.write(REPO_CONTENT.format(repo_url, AUTHORITY_CERT, PRIVATE_KEY, AGENT_CERT))
+    tmp.write(
+        REPO_CONTENT.format(repo_url, AUTHORITY_CERT, PRIVATE_KEY, AGENT_CERT))
 
     tmp.flush()
     launch_command('cp %s %s' % (tmp.name, REPO_PATH))
     os.chmod(REPO_PATH, 0644)
 
 
 def install_agent():
-    return launch_command('yum install -y --enablerepo=%s %s' % ('{repo_names}', repo_packages))
+    return launch_command('yum install -y --enablerepo=%s %s' %
+                          ('{repo_names}', repo_packages))
 
 
 def configure_server():
-    launch_command('chroma-agent set_server_url --url %s' % base_url + 'agent/')
-    launch_command("chroma-agent set_profile --profile_json '%s'" % profile_json)
+    launch_command(
+        'chroma-agent set_server_url --url %s' % base_url + 'agent/')
+    launch_command(
+        "chroma-agent set_profile --profile_json '%s'" % profile_json)
 
 
-def start_agent():
-    output = launch_command("systemctl enable chroma-agent")
-    output = launch_command("systemctl start chroma-agent")
+def start_services():
+    launch_command("systemctl enable chroma-agent.service")
+    launch_command("systemctl start iml-storage-server.target")
 
 
 def kill_zombies():
+    try:
+        # attempt to stop a pre-installed instance
+        launch_command("systemctl stop iml-storage-server.target")
+    except Exception:
+        # Will fail if not pre-installed
+        pass
```

Comment:
This says it should fail but it says `pass`?

Reply:
yes, trying to stop the target will fail if it's not installed.

In that case it will raise an exception, but we want to continue because there was nothing there anyway.